integration/copy_test.go

```diff
@@ -160,6 +163,56 @@ func (s *CopySuite) TestCopySimple(c *check.C) {
 
 }
 
+func (s *CopySuite) TestCopyEncryption(c *check.C) {
+
+	dir1, err := ioutil.TempDir("", "copy-1")
+	c.Assert(err, check.IsNil)
+	defer os.RemoveAll(dir1)
+	dir2, err := ioutil.TempDir("", "copy-2")
+	c.Assert(err, check.IsNil)
+	defer os.RemoveAll(dir2)
+	dir3, err := ioutil.TempDir("", "copy-3")
+	c.Assert(err, check.IsNil)
+	defer os.RemoveAll(dir3)
+	dir4, err := ioutil.TempDir("", "copy-4")
+	c.Assert(err, check.IsNil)
+	defer os.RemoveAll(dir4)
```

Comment:
Can we name those variables as something more descriptive? E.g., `dir1` as `unencryptedDir`? It make the tests easier to understand and ultimately easier to maintain in the future.